{
    "_type": "pyessv._model.term", 
    "canonicalName": "qc-flag-channel-3-failure", 
    "createDate": "2018-03-09 00:00:00+00:00", 
    "data": {
        "dimension": "time", 
        "flagMeanings": "not_used good_data  bad_data_channel_failure suspect_data_time_stamp_error", 
        "flagValues": "0b,1b, 2b, 3b", 
        "longName": "Data Quality flag: Receiver channel 3 failed", 
        "name": "qc_flag_channel_3_failure", 
        "type": "byte", 
        "units": "1"
    }, 
    "idx": 16, 
    "label": "qc_flag_channel_3_failure", 
    "rawName": "qc_flag_channel_3_failure", 
    "status": "pending", 
    "uid": "55e767eb-a36f-4f62-b602-1f4dd0f361d9"
}